package.json

```diff
@@ -63,7 +63,7 @@
   },
   "dependencies": {
     "cssnano": "^5.1.8",
-    "jest-worker": "^27.5.1",
+    "jest-worker": "^29.0.3",
```

Comment:
Please remove this.

Reply:
done

Reply:
this is required, otherwise [lint job is failing.](https://github.com/webpack-contrib/css-minimizer-webpack-plugin/actions/runs/3075582245/jobs/4969110107#step:6:28)

Reply:
Oh, okay 👍🏻